bindings/python/src/OpenSpaceToolkitPhysicsPy/Environment/Objects/CelestialBodies/Earth.cpp

```diff
@@ -109,6 +113,18 @@ inline void                     OpenSpaceToolkitPhysicsPy_Environment_Objects_Ce
 
```

Comment:
These models are not properly attached to the class and can't be used in the bindings